deploy/post_deploy_testing/cypress/integration/03d_browse_views_files_selection.js

```diff
@@ -1,154 +1,191 @@
-
-
-function checkUncheckFileCheckbox(origSelectedCount, $checkBox){
-    return cy.wrap($checkBox).scrollToCenterElement().uncheck({ 'force' : true }).end()
-        .getDownloadButton().invoke('text').then((downloadButtonTextContent)=>{
+function checkUncheckFileCheckbox(origSelectedCount, $checkBox) {
+    return cy.wrap($checkBox).scrollToCenterElement().uncheck({ 'force': true }).end()
+        .getDownloadButton().invoke('text').then((downloadButtonTextContent) => {
             let nextSelectedCount = downloadButtonTextContent.match(/\d/g);
             nextSelectedCount = parseInt(nextSelectedCount.join(''));
             expect(nextSelectedCount).to.be.lessThan(origSelectedCount);
             return cy.getSelectAllFilesButton().should('not.contain', "Deselect All");
-        }).end().wrap($checkBox).check({ 'force' : true }).end()
-        .getDownloadButton().invoke('text').then((downloadButtonTextContent)=>{
+        }).end().wrap($checkBox).check({ 'force': true }).end()
+        .getDownloadButton().invoke('text').then((downloadButtonTextContent) => {
             let nextSelectedCount = downloadButtonTextContent.match(/\d/g);
             nextSelectedCount = parseInt(nextSelectedCount.join(''));
             expect(nextSelectedCount).to.equal(origSelectedCount);
             return cy.getSelectAllFilesButton().should('contain', "Deselect All");
         });
-}
-
 
+}
+let fileTypeSelectedTotalCount = 0;
+function fileTypeFilterCheckFileCheckbox(origSelectedCount, $checkBox) {
+    return cy.wrap($checkBox).scrollToCenterElement().check({ 'force': true }).end()
+        .getDownloadButton().invoke('text').then((downloadButtonTextContent) => {
+            const nextSelectedCount = downloadButtonTextContent.match(/\d/g);
+            fileTypeSelectedTotalCount = parseInt(nextSelectedCount.join(''));
+        });
+}
+function fileTypeFilterUnCheckFileCheckbox($checkBox) {
+    return cy.wrap($checkBox).scrollToCenterElement().uncheck({ 'force': true }).end();
+}
 describe('Browse Views - Files Selection', function () {
 
-    context('Ensure "Select All Files" & related features are functional.', function(){
+    context('Ensure "Select All Files" & related features are functional.', function () {
 
-        before(function(){
-            cy.visit('/browse/?award.project=4DN&experimentset_type=replicate&type=ExperimentSetReplicate');
+        before(function () {
+            cy.visit('/browse/?award.project=4DN&experimentset_type=replicate&type=ExperimentSetReplicate&experiments_in_set.experiment_type.display_title=CUT%26RUN');
         });
 
-        it('We have some counts in QuickInfoBar to compare selected file counts against.', function(){
+        it.skip('We have some counts in QuickInfoBar to compare selected file counts against.', function () {
```

Comment:
Maybe remove this `.skip` (if was just left in after testing and forgot to revert)